org-journal.el

```diff
@@ -196,6 +196,11 @@ saves/opens these journal entries, emacs asks a user passphrase
 to encrypt/decrypt it."
   :type 'boolean)
 
+(defcustom org-journal-encrypt-journals nil
```

Comment:
I feel this should be `org-journal-encrypt-journal` (no plural), since org-journal is keeping one journal (but many entries).